dft/src/butterflies.rs

```diff
@@ -57,17 +103,39 @@ pub trait Butterfly<F: Field>: Copy + Send + Sync {
     }
 }
 
+/// DIF (Decimation-In-Frequency) butterfly operation.
+///
+/// Used in the *output-ordering* variant of NTT.
+/// This butterfly computes:
+///
+///   output_1 = x1 + x2
+///   output_2 = (x1 - x2) * twiddle
+///
+/// The twiddle factor is applied after subtraction.
+/// Suitable for DIF-style recursive transforms.
 #[derive(Copy, Clone)]
 pub struct DifButterfly<F>(pub F);
+
 impl<F: Field> Butterfly<F> for DifButterfly<F> {
     #[inline]
     fn apply<PF: PackedField<Scalar = F>>(&self, x_1: PF, x_2: PF) -> (PF, PF) {
         (x_1 + x_2, (x_1 - x_2) * self.0)
     }
 }
 
+/// DIT (Decimation-In-Time) butterfly operation.
+///
+/// Used in the *input-ordering* variant of NTT/FFT.
+/// This butterfly computes:
+///
+///   output_1 = x1 + x2 * twiddle
+///   output_2 = x1 - x2 * twiddle
+///
```

Comment:
```suggestion
/// ```text
///   output_1 = x1 + x2 * twiddle
///   output_2 = x1 - x2 * twiddle
/// ```
```